### ISSUE TITLE
Error in "Your first Aragon app" tutorial

### ISSUE THREAD

**Issue by sohkai** [aragon/aragon-cli#274]
@yutarond commented on [Fri Nov 16 2018](https://github.com/aragon/hack/issues/55)

I tried following this ["Your first Aragon app" tutorial](https://hack.aragon.org/docs/tutorial.html). When I run `aragon run` command, got an error bellow:

```
 ❯ Start a local Ethereum network
   ⠴ Setting up a new chain from latest Aragon snapshot
     Starting a local chain from snapshot
   Check IPFS
   Publish app to APM
   Create DAO
   Open DAO

/Users/yutaro/.nodebrew/node/v9.5.0/lib/node_modules/@aragon/cli/node_modules/ganache-core/node_modules/level-codec/index.js:44
  return this._valueEncoding(opts).decode(value)
                                   ^
SyntaxError: Unexpected token o in JSON at position 1
    at Object.parse [as decode] (<anonymous>)
    at Codec.decodeValue (/Users/yutaro/.nodebrew/node/v9.5.0/lib/node_modules/@aragon/cli/node_modules/ganache-core/node_modules/level-codec/index.js:44:36)
    at /Users/yutaro/.nodebrew/node/v9.5.0/lib/node_modules/@aragon/cli/node_modules/ganache-core/node_modules/level-sublevel/nut.js:122:34
    at /Users/yutaro/.nodebrew/node/v9.5.0/lib/node_modules/@aragon/cli/node_modules/ganache-core/node_modules/encoding-down/index.js:57:5
    at /Users/yutaro/.nodebrew/node/v9.5.0/lib/node_modules/@aragon/cli/node_modules/ganache-core/node_modules/cachedown/index.js:61:5
    at /Users/yutaro/.nodebrew/node/v9.5.0/lib/node_modules/@aragon/cli/node_modules/ganache-core/lib/database/filedown.js:28:5
    at FSReqWrap.readFileAfterClose [as oncomplete] (fs.js:528:3)
```

**Comment by sohkai** [comment]
@yutarond I moved this repo to the CLI, since it appears that's where the issue is.

Seems like it's something to do with ganache, which is odd. Maybe there's something wrong with the new aragen snapshot @izqui?

**Comment by macor161** [comment]
I submitted a pull request that uses `ganache-core v2.2.1`.
This is a temporary fix until we get to know exactly what in the new `ganache-core` version is causing the bug.

**Comment by bingen** [comment]
Yes, indeed that seems to be the problem, The exact change that is breaking it is this one:
https://github.com/trufflesuite/ganache-core/commit/22e3af697dcd1ee68e7f671ad3bdc128b7c5f1bc#diff-f7055d87f223a4d22d9d739ca35e954aR36
In particular, it seems to be the addition of `encoding-down`.

**Comment by bingen** [comment]
This is fixed temporarily in #280. 
For a more stable solution, I opened a PR in `ganache-core` repo, not sure if that's the best way to fix it, but their tests are passing and our `cli` is working with it: 
https://github.com/trufflesuite/ganache-core/pull/239

**Comment by sohkai** [comment]
@galactusss @0x6431346e It looks like with the newest [ganache-core@2.3.2](https://github.com/trufflesuite/ganache-core/releases/tag/v2.3.2) the snapshot regression should be fixed. We should try unpinning and making sure everything works :).

**Comment by bingen** [comment]
Oh, I just pushed PR #302 but I still want to test it thoroughly. You can give it try though.

**Comment by galactusss** [comment]
I tested all the steps on the tutorial and it seem the error was fixed with #302 PR 
should close this? Or you want to test a bit more before @bingen @0x6431346e

**Comment by bingen** [comment]
It's fine for me if you tested it. I just merged #302. Thanks @galactusss !
